src/test/java/Page_Objects/SongLanguagesPage.java

```diff
@@ -20,4 +23,29 @@ public SongLanguagesPage(WebDriver driver) {
     public void continueAndGoToSongList() {
         continueButton.click();
     }
+
+    public WebElement getLanguageElement(String language) {
+        return driver.findElement(By.cssSelector("[data-test=lang-" + language + "]"));
+    }
+
+    public WebElement getSongLanguageCheckbox(String language) {
+        return getLanguageElement(language).findElement(By.cssSelector("svg"));
+    }
+
+    public void ensureSongLanguageIsSelected(String language) {
+        if(getSongLanguageCheckbox(language).getAttribute("data-testid").equals("CheckBoxOutlineBlankIcon")) {
+            getSongLanguageCheckbox(language).click();
+        }
+    }
+
+    public void ensureAllSongLanguagesAreSelected() {
+
+        List<WebElement> languages = driver.findElements(By.cssSelector("[data-test^='lang-'] svg"));
+
+        for (WebElement language : languages) {
+            if (language.getAttribute("data-testid").equals("CheckBoxOutlineBlankIcon")) {

```

Comment:
this check could be moved to a separate method (`isLanguageSelected`) as it's used in two methods - this one and `ensureSongLanguageIsSelected`